GitHubContents.ps1

```diff
@@ -358,7 +385,14 @@ filter Set-GitHubContent
 
     $uriFragment = "/repos/$OwnerName/$RepositoryName/contents/$Path"
 
-    $encodedContent = [Convert]::ToBase64String([System.Text.Encoding]::UTF8.GetBytes($Content))
+    if (-not [String]::IsNullOrWhitespace($ContentPath))
```

Comment:
Given that what you really mean here is "if our parameter set included `ContentPath`, let's go in here", it would be more clear (and correct) if you did this.

```suggestion
    if ($PSBoundParameters.ContainsKey('ContentPath')
```

Otherwise, what you're saying is, if `$ContentPath` only has whitespace, let's use `$Content` instead, but we won't have that at all because that wasn't in our parameter set.

Reply:
`$PSBoundParameters` Thank you, you are teaching me some PowerShell now.